index.bs

```diff
@@ -2527,6 +2530,31 @@ needs.
 </div>
 
 
+## Feature Policy integration ## {#sctn-feature-policy}
+
+This specification defines a [=policy-controlled feature=] identified by
+the feature policy token "<code><dfn data-lt="webauthn-feature">webauthn</dfn></code>".
+Its [=default allowlist=] is '<code>self</code>'. [[!Feature-Policy]]
+
+A {{Document}}'s [=Document/feature policy=] determines whether any content in that <a href="https://html.spec.whatwg.org/multipage/dom.html#documents">document</a> is allowed
+to successfully invoke the [=Web Authentication API=], i.e., via
+<code><a idl for="CredentialsContainer" lt="create()">navigator.credentials.create({publicKey:..., ...})</a></code> and
+<code><a idl for="CredentialsContainer" lt="get()">navigator.credentials.get({publicKey:..., ...})</a></code>.
+If disabled in any document, no content in the document will be [=allowed to use=]
+the foregoing methods: attempting to do so will throw an error.
```

Comment:
Not just the foregoing methods, but any Webauthn method, no?

Reply:
Ah, good question, thx. Presently, [FP](https://www.w3.org/TR/feature-policy/) is integrated into only the `[[Create]]()` and `[[DiscoverFrom...]]()` internal methods.  I neglected `IsUVPAA()`, will look into it.   

I am guessing we do not need to address `PublicKeyCredential.getClientExtensionResults()` because if it was not minted by  the `[[Create]]()` and `[[DiscoverFrom...]]()` internal methods (ie some random JS code does a `new PublicKeyCredential`, there's nothing to return from `getClientExtensionResults()`.  And such random code would be able to mint a `PublicKeyCredential` regardless of feature-policy, yes?  

Similar argument for `AuthenticatorAttestationResponse.getTransports()`.

I do not see any other WebAuthn API-specific methods... ?

Reply:
> I am guessing we do not need to address `PublicKeyCredential.getClientExtensionResults()` [&hellip;] similar argument for `AuthenticatorAttestationResponse.getTransports()`

Yep, I think that's correct.